core/src/main/java/io/cucumber/core/api/event/TestCaseStarted.java

```diff
@@ -2,10 +2,16 @@
 
 public final class TestCaseStarted extends TestCaseEvent {
     public final TestCase testCase;
+    private final long elapsedTimiMillis; 
 
-    public TestCaseStarted(Long timeStamp, TestCase testCase) {
+    public TestCaseStarted(Long timeStamp, Long elapsedTimiMillis, TestCase testCase) {
         super(timeStamp, testCase);
         this.testCase = testCase;
+        this.elapsedTimiMillis = elapsedTimiMillis;
+    }
+
+    public long getElapsedTimiMillis() {
+        return elapsedTimiMillis;
```

Comment:
I don't think `elapsedTimiMillis` is quite right. Perhaps we can rename this to `timeStampMillis`.